src/gtfs/read.rs

```diff
@@ -2645,14 +2645,11 @@ mod tests {
                 vec![
                     Geometry {
                         id: "1".to_string(),
-                        geometry: GeoGeometry::LineString(LineString(vec![
-                            Point::new(1.1, 2.2),
-                            Point::new(3.3, 4.4),
-                        ])),
+                        geometry: GeoGeometry::LineString(vec![(1.1, 2.2), (3.3, 4.4),].into()),
```

Comment:
this is completely valid, but just for your information, you could also have used the [new macros to reduce the xy/yx ambiguity](https://github.com/georust/geo/pull/352):
`geometry: GeoGeometry::line_string![(x: 1.1, y: 2.2), (x: 3.3, y:4.4)]`

Reply:
not in 0.4.2